### PR TITLE
feat: add stacks-gen

### PR DESCRIPTION
**Pull request by psq**
This adds a reference to [stacks-gen](https://github.com/psq/stacks-gen) to the accounts creation page

### CODE REVIEW

**Review by agraebe — APPROVED** [review]
LGTM, thanks @psq

**Review by aulneau — CHANGES_REQUESTED** [review]

File: src/pages/stacks-blockchain/accounts.md [hunk 1]
```diff
@@ -84,6 +84,24 @@ const stacksAddress = getAddressFromPrivateKey(
 );
 ```
 
+A second alternative would be to use [stacks-gen](https://github.com/psq/stacks-gen).  This tool will generate all the keys you need in one place, including the values needed for calling the stacking contract, and also a WIF key for use with `bitcoind`.
+
+```
+npx -q stacks-gen sk --testnet
```
[the remark this answers is not in the document]
Comment (psq) [reply src/pages/stacks-blockchain/accounts.md:90]: @aulneau I did not include it as this was not included on the mining page either: https://github.com/blockstack/docs/blob/70fad9f0e5072eb41fb4ca7a6ba8c6c3eed735bf/src/pages/mining.md#L28\

As someone who really does not like to install things globally, I find npx more suitable.  I'll add explanations on how to get `npx` installed.

[the remark this answers is not in the document]
Comment (psq) [reply src/pages/stacks-blockchain/accounts.md:97]: addressed by c6596f9

[the remark this answers is not in the document]
Comment (aulneau) [reply src/pages/stacks-blockchain/accounts.md:97]: Can you link to npx so folks can learn more about it? I use it too, but there are risks as it runs any arbitrary code immediately, and some people might not prefer that.

[the remark this answers is not in the document]
Comment (psq) [reply src/pages/stacks-blockchain/accounts.md:97]: added a link and a small blurb in 68b8ece32dbbadb150730cfc676d51a7d81716b9

[the remark this answers is not in the document]
Comment (psq) [reply src/pages/stacks-blockchain/accounts.md:97]: @aulneau could you check whether my latest change addresses your comment?

**Review by aulneau — APPROVED** [review]

**Comment by psq** [comment]
note: I don't have merge access, so someone will need to merge that in... Thanks!

**Pull request merged**